.gitattributes

```diff
@@ -0,0 +1,11 @@
+* text=auto
+
+/spec export-ignore
+/.gitignore export-ignore
+/.gitattributes export-ignore
+/.travis.yml export-ignore
+/CHANGELOG.md export-ignore
```

Comment:
Atom provides functionality to view the `CHANGELOG.md` in the Settings, this shouldn't be excluded.